cmd/influxd/server_integration_test.go

```diff
@@ -197,8 +198,13 @@ func query(t *testing.T, nodes Cluster, urlDb, query, expected string) (string,
 			t.Fatalf("Couldn't read body of response: %s", err.Error())
 		}
 
-		if expected != string(body) {
+		if expected != "" && expected != string(body) {
 			return string(body), false
+		} else if expectPattern != "" {
```

Comment:
I've felt like we've need regex matching for a while in the suite, so it's good to see it.